format/index.md

```diff
@@ -52,761 +58,508 @@ called out below.
-GET /posts?include=author&sort[posts]=-created,title&sort[people]=name
-```
+> Note: JSON API is agnostic about the pagination strategy used by a server.
+Effective pagination strategies include (but are not limited to):
+page-based, offset-based, and cursor-based. The `page` query parameter can
+be used as a basis for any of these strategies. For example, a page-based
+strategy might use query parameters such as `page[number]` and `page[size]`,
+an offset-based strategy might use `page[offset]` and `page[limit]`, while a
+cursor-based strategy might use `page[cursor]`.
 
-If no sort order is specified, or if the endpoint does not support use of either
-`sort` or `sort[TYPE]`, then the endpoint **SHOULD** return resource objects
-sorted with a repeatable algorithm. In other words, resources **SHOULD** always
-be returned in the same order, even if the sort criteria aren't specified.
+### Filtering <a href="#fetching-filtering" id="fetching-filtering" class="headerlink"></a>
 
-Note: `sort` and `sort[TYPE]` can not be mixed. If the latter format is used,
-then it **MUST** be used for the primary resource type as well.
+The `filter` query parameter is reserved for servers to use for filtering data.
 
+> Note: JSON API is agnostic about the filtering strategies supported by a
+server. The `filter` query parameter can be used as the basis for any number of
+filtering strategies.
 
 ## Creating, Updating and Deleting Resources <a href="#crud" id="crud" class="headerlink"></a>
 
-A server **MAY** allow resources that can be fetched to also be created,
-modified and deleted.
+A server **MAY** allow resources of a given type to be created. It **MAY**
+also allow existing resources to be modified or deleted.
 
-A server **MAY** allow multiple resources to be updated in a single request, as
-discussed below. Updates to multiple resources **MUST** completely succeed or
-fail. No partial updates are allowed.
+Any requests that contain content **MUST** include a `Content-Type` header
+whose value is `application/vnd.api+json`. This header value **MUST** also
+include media type extensions relevant to the request.
 
-Any requests that contain content **MUST** include a `Content-Type` header whose
-value is `application/vnd.api+json`.
+A request **MUST** completely succeed or fail (in a single "transaction"). No
+partial updates are allowed.
 
-### Creating Resources <a href="#crud-creating-resources" id="crud-creating-resources" class="headerlink"></a>
+### Creating Resources <a href="#crud-creating" id="crud-creating" class="headerlink"></a>
 
-A server that supports creating resources **MUST** support creating individual
-resources and **MAY** optionally support creating multiple resources in a single
-request.
+A resource can be created by sending a `POST` request to a URL that represents
+a collection of resources. The request **MUST** include a single resource object
+as primary data. The resource object **MUST** contain at least a `type` member.
 
```

Comment:
>> The request **MUST** include a single resource object-request as primary data.

This, and similar statements in updating and deleting, is violated by the bulk extension. If that violation is allowable (and it should be, at least for official extensions), then perhaps there should be a statement to that effect in /extensions/index.md